### PR TITLE
Searchbar and single Product page

### PR DESCRIPTION
**Pull request by gadgetfather**
- user can search for product
- Single Product page has been added

### Preview
searchbar

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -35,60 +38,69 @@ export function Navbar() {
...
+          <Link to="/" className="brand-name">
+            <h1>Manazone</h1>
+          </Link>
+          <Searchbar
+            searchValue={searchValue}
+            setSearchValue={setSearchValue}
+          />
+          {/* <div className="search-bar-desktop">
           <i className="fas fa-search"></i>
           <input type="search" />
```
Comment (kart-c) [inline src/components/Navbar/Navbar.jsx]: you can remove this if not needed

File: src/components/ProductCard/ProductCard.jsx [hunk 2]
```diff
@@ -10,30 +10,40 @@ export function ProductCard(props) {
...
 
-  const { title, image, price, categoryName, stars, badge, id, _id } = props;
+  const { title, image, price, categoryName, stars, badge, id, _id, data_id } =
+    props;
   const { addToCart, cartData } = useCart();
   const { addToWishlist, wishlistData, removeFromWishlist } = useWishlist();
-
   const handleAddToCart = (props) => {
+    e.stopPropagation();
+
```
Comment (kart-c) [inline src/components/ProductCard/ProductCard.jsx]: you need to accept `e` as a parameter. This function is throwing an error

File: src/components/ProductCard/ProductCard.jsx [hunk 3]
```diff
@@ -10,30 +10,40 @@ export function ProductCard(props) {
...
     if (token) {
       addToCart(props);
     } else {
       navigate("/login");
     }
   };
   const handleAddToWishlist = (data) => {
+    e.stopPropagation();
+
```
Comment (kart-c) [inline src/components/ProductCard/ProductCard.jsx]: need to accept `e` as a parameter here as well

File: src/components/ProductCard/ProductCard.jsx (hunk 2 above)
Comment (gadgetfather) [reply src/components/ProductCard/ProductCard.jsx]: Fixed the bug. Thanks for pointing it out !

**Review by shristabaruah — APPROVED** [review]

**Pull request merged**